pages/prerequisites/CKLS.md

```diff
@@ -14,8 +14,8 @@ CrossKnowledge Learning Suite is a web-based application needing the following e
 
 Screen resolution|1024*768
 Sound card|Recommended (speakers/headphones are required for most content)
-RAM|2 GB (4GB preferred, depending on the OS)
-Operating system|Microsoft Windows 8.1 (Desktop mode only) or 10, Apple macOS (10.13+)
+RAM|4GB+
```

Comment:
we have to change the last_updated date in the header of this file

Reply:
Ok now.